lambdas/service/app.py

```diff
@@ -139,8 +139,47 @@ def version():
     }
 
 
+def validate_repository_search(incoming_params, **kwargs):
+    valid_params = {
+        'filters': str,
+        'order': str,
+        'search_after': str,
+        'search_after_uid': str,
+        'search_before': str,
+        'search_before_uid': str,
+        'size': int,
+        'sort': str,
+        **kwargs
+    }
+    validate_params(incoming_params, **valid_params)
+
+
+def validate_params(query_params, allow_extra_params=False, **kwargs):
+    """
+    Validates incoming request query parameters for web-service API.
+    :param query_params: Incoming parameters to be validated.
+    :param allow_extra_params: Regards the strictness by which the incoming parameters should be validated. When False,
```

Comment:
Regards isn't the right word here. Maybe "configure"?
```suggestion
    :param allow_extra_params: When False,
```